test/integration/sagemaker/test_distributed_operations.py

```diff
@@ -31,12 +33,33 @@ def test_dist_operations_gpu(sagemaker_session, instance_type, ecr_image, dist_g
 
 @pytest.mark.skip_cpu
 def test_dist_operations_multi_gpu(sagemaker_session, ecr_image, dist_gpu_backend):
-    instance_type = 'ml.p3.8xlarge'
-    _test_dist_operations(sagemaker_session, ecr_image, instance_type, dist_gpu_backend, 1)
+    _test_dist_operations(sagemaker_session, ecr_image, MULTI_GPU_INSTANCE, dist_gpu_backend, 1)
+
+
+@pytest.mark.skip_cpu
+def test_dist_operations_fastai_gpu(sagemaker_session, ecr_image, py_version):
+    if py_version != PYTHON3:
+        print('Skipping the test because fastai supports >= Python 3.6.')
+        return
+
+    with timeout(minutes=DEFAULT_TIMEOUT):
+        pytorch = PytorchTestEstimator(entry_point='train_cifar.py',
+                                       source_dir=os.path.join(fastai_path, 'cifar'),
+                                       role='SageMakerRole',
+                                       train_instance_count=1,
+                                       train_instance_type=MULTI_GPU_INSTANCE,
+                                       sagemaker_session=sagemaker_session,
+                                       docker_image_uri=ecr_image)
+        pytorch.sagemaker_session.default_bucket()
+        training_nput = pytorch.sagemaker_session.upload_data(
+            path=os.path.join(fastai_path, 'cifar_tiny', 'training'),
+            key_prefix='pytorch/distributed_operations'
+        )
+        pytorch.fit({'training': training_nput})
```

Comment:
Should we assert something here? Verify the model or any artifact is saved?

Reply:
We can, doesn't really change anything for that case (because the main assertion is that it doesn't fail that's why we didn't assert in majority of our tests before).
But i guess now when instance type is defined outside of test it will be more useful to make sure we didn't skip the test because of using wrong instance type.